packages/e2e-test-utils/src/open-global-inserter.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Opens the global inserter
+ */
+export async function openGlobalInserter() {
```

Comment:
Can we make it more specific?
```suggestion
export async function openGlobalBlockInserter() {
```